AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1499,6 +1499,8 @@ oneplus.com,tickets.ua,woman.ru,elle.ru,marieclaire.ru,europaplus.ru,krossovki.c
 !
 tochka.net##.subscribe_widget
 adidas.com.tr##div[class^="footer_newsletter"]
+blog.remarkable.com#?#.screenContent > div.js-collectionStream > div.streamItem:has(> div.u-marginAuto > div.u-marginAuto > div.u-marginBottom20 > a[href="https://mailchi.mp/mail.remarkable.com/subscribe_to_newsletter"])
+remarkable.com#?#main > div[style] > article[class^="business-module"]:has(> div[class^="business-module"] > h2:contains(Stay in the loop))
```

Comment:
cannot find where these rules must be applied.
And share buttons at https://blog.remarkable.com/introducing-read-on-remarkable-6981369b6399

Reply:
![Screenshot 2020-09-02 at 14 56 08](https://user-images.githubusercontent.com/18536328/91978260-7e357d00-ed2c-11ea-9203-ea74e2b935d8.png)
![Screenshot 2020-09-02 at 14 53 53](https://user-images.githubusercontent.com/18536328/91978266-7fff4080-ed2c-11ea-8d8a-c9be67f8dd7e.png)